classes/models/LengowGender.php

```diff
@@ -84,12 +94,16 @@ public static function getGender($name)
         if (in_array($name, self::$currentFemale, true)) {
```

Comment:
Can all of this strings be in lowercase to have only one of the same word ?
And check it like `in_array(strtolower($name), self::$currentFemale, true)`